### PR TITLE
Fix broken amp-fx="fly-in-{left|right}" for block positioned elements

### PR DESCRIPTION
**Pull request by nainar**
We need to set the `position` property of the element here for the `left` property to have any affect. See: https://developer.mozilla.org/en/docs/Web/CSS/left 

Fixes: #19237

### CODE REVIEW

**Comment by nainar** [comment]
@sparhami made the change to only override position if not already set

File: extensions/amp-fx-collection/0.1/providers/amp-fx-presets.js [hunk 1]
```diff
@@ -82,10 +82,13 @@ export const Presets = {
               const style = computedStyle(fxElement.getAmpDoc().win,
                   fxElement.getElement());
               const topAsLength = style.top == 'auto' ? '0px' : style.top;
```
Comment (sparhami) [inline extensions/amp-fx-collection/0.1/providers/amp-fx-presets.js]: lets change this to `===` to match the others.

File: extensions/amp-fx-collection/0.1/providers/amp-fx-presets.js [hunk 2]
```diff
@@ -218,11 +228,14 @@ export const Presets = {
             fxElement.getElement(), function() {
               const style = computedStyle(fxElement.getAmpDoc().win,
                   fxElement.getElement());
-              const topAsLength = style.top == 'auto' ? '0px' : style.top;
+              const topAsLength = style.top === 'auto' ? '0px' : style.top;
+              const positionKeyword =
+                style.position !== 'static' ? 'relative' : 'static';
```
Comment (sparhami) [inline extensions/amp-fx-collection/0.1/providers/amp-fx-presets.js]: shouldn't this be the same as before?

```javascript
const positionKeyword =
    style.position === 'static' ? 'relative' : style.position;
```

File: extensions/amp-fx-collection/0.1/providers/amp-fx-presets.js [hunk 3]
```diff
@@ -82,10 +82,13 @@ export const Presets = {
               const style = computedStyle(fxElement.getAmpDoc().win,
                   fxElement.getElement());
               const topAsLength = style.top == 'auto' ? '0px' : style.top;
+              const positionKeyword =
```
Comment (sparhami) [inline extensions/amp-fx-collection/0.1/providers/amp-fx-presets.js]: shouldn't this be:

```javascript
const positionKeyword = style.position === 'static' ? 'relative' : style.position;
```

Comment (nainar) [reply extensions/amp-fx-collection/0.1/providers/amp-fx-presets.js]: Done

File: extensions/amp-fx-collection/0.1/providers/amp-fx-presets.js (hunk 1 above)
Comment (nainar) [reply extensions/amp-fx-collection/0.1/providers/amp-fx-presets.js]: Done

File: extensions/amp-fx-collection/0.1/providers/amp-fx-presets.js (hunk 2 above)
Comment (nainar) [reply extensions/amp-fx-collection/0.1/providers/amp-fx-presets.js]: Done

**Comment by nainar** [comment]
@sparhami take a look?

File: extensions/amp-fx-collection/0.1/test/integration/test-amp-fx-fly-in.js [hunk 4]
```diff
@@ -16,7 +16,7 @@
 
 import {isExperimentOn, toggleExperiment} from '../../../../../src/experiments';
 
-describe.configure().run('amp-fx-collection', function() {
+describe.skip('amp-fx-collection', function() {
```
Comment (aghassemi) [inline extensions/amp-fx-collection/0.1/test/integration/test-amp-fx-fly-in.js]: @nainar was this `skip` intentional?

**Pull request merged**